packages/plugin-core/src/abTests.ts

```diff
@@ -0,0 +1,25 @@
+import { ABTest } from "@dendronhq/common-server";
+
+// ^xi5t1r2j51ot
+
+export enum UpgradeToastOrViewTestEnum {
+  upgradeToast = "upgradeToast",
+  upgradeView = "upgradeView",
+}
+
+export const UPGRADE_TOAST_OR_VIEW_TEST = new ABTest("UpgradeToastOrViewTest", [
```

Comment:
maybe having a quick 1-2 liner description of what the experiment is about would be helpful.